resources/platformsh/common/3.3.x-dev/.platform/services.yaml

```diff
@@ -1,4 +1,4 @@
-# Default settings in order to set up eZ Platform demo install on Platform.sh dev instances
+# Default settings in order to set up Ibexa DXP install on Ibexa Cloud dev instances
```

Comment:
IMHO it should be "installation". Proof reading ping @ibexa/documentation

Reply:
Fixed in https://github.com/ibexa/post-install/pull/12/commits/e7064a8a31677f6d632593394982195ebba6207f